controllers/admins.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const admins = require('../data/admin.json');
+
+//Get all admins
+
+const getAdmins = (req, res) => {
+    res.status(200).json(
+        res.json(admins)
```

Comment:
you already are returning a .json with `res.status(200).json(`. 
Using `res.json(admins)` into it is not necessary.